### PR TITLE
feat: upgrade kroma mainnet and kroma sepolia to v1.3.3

### PR DESCRIPTION
**Pull request by 0xHansLee**
This includes both Kroma mainnet and Kroma sepolia for `v1.3.3` upgrade

### CODE REVIEW

File: upgrades/v1.3.3.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Kroma `v1.3.2` Upgrade Instructions
```
Comment (kangsorang) [inline upgrades/v1.3.3.md]:
```suggestion
# Kroma `v1.3.3` Upgrade Instructions
```

**Review by kangsorang — APPROVED** [review]

**Comment by seolaoh** [comment]
Don't need to apply [this change](https://github.com/kroma-network/kroma-up/pull/29/files#diff-e5e759c858fdef3fe1fa140afc4987e0ba85bed7b8151c745801d09ecf1be5eb)?

File: upgrades/v1.3.3.md (hunk 1 above)
Comment (0xHansLee) [reply upgrades/v1.3.3.md]: Fixed in [8539d2f](https://github.com/kroma-network/kroma-up/pull/32/commits/8539d2f0c7c8872a7c9930d611443cf4508d4b13)

**Comment by 0xHansLee** [comment]
> Don't need to apply [this change](https://github.com/kroma-network/kroma-up/pull/29/files#diff-e5e759c858fdef3fe1fa140afc4987e0ba85bed7b8151c745801d09ecf1be5eb)?

I applied the previous change. Thank you for pointing out.

**Review by seolaoh — APPROVED** [review]

File: upgrades/v1.3.3.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+### Common
+
+You need to update your `.env` file.
+
+First, update the tag of `kroma-node` and `kroma-validator` docker image.
+```
+IMAGE_TAG__KROMA_GETH=v0.4.5
+IMAGE_TAG__KROMA_NODE=v1.3.3
+IMAGE_TAG__KROMA_VALIDATOR=v1.3.3
+```
```
Comment (seolaoh) [inline upgrades/v1.3.3.md:30]: How about adding about the change in `geth.env`?

**Pull request merged**